CWE/events/algeria.txt

```diff
@@ -206,8 +206,12 @@ country_event = {
 		war = no
 		is_vassal = no
 		
+		government = theocracy
+		ruling_party_ideology = traditionalist
```

Comment:
Should also fire event 800050 to set the appropriate reforms

Reply:
Fixed it.